pkg/assembler/graphql/schema/osv.graphql

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2023 The GUAC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This is experimental and might change in the future!
+
+# Defines a GraphQL schema for the osv trie/tree. It contains 
+# id associated with vulnerability (osv id)
+"""
+OSV represents Open Source Vulnerability . It contains a OSV ID.
+
+"""
+type OSV {
+  osv_id: [OSV_ID!]!
```

Comment:
is there a styling guide for GraphQL? I found this, but not sure since it may be arbitrary (https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions).. It does look like we've been following PascalCase for types and camelCase for variables?

@mihaimaruseac

Reply:
Similar question for acronym naming in variables. All caps is usually what's done in Go.

Reply:
I will change the `osv_id` to camelCase